Scripts/script-WordParser.yml

```diff
@@ -0,0 +1,49 @@
+commonfields:
+  id: WordParser
+  version: -1
+name: WordParser
+comment: Takes an input docx file (entryID) as an input and saves an output text file (file entry) with the original file's contents.
+script: |
+  from docx import Document
+  from docx.opc.exceptions import PackageNotFoundError
+
+  max_file_size = 1024 ** 2
```

Comment:
Where do you use it? why this number?

Reply:
No reason, other attempts leftovers. Removed.